### ISSUE TITLE
[CST] Details V2: Closed claim Status tab 

### ISSUE THREAD

**Issue by skylerschain** [department-of-veterans-affairs/va.gov-team#77239]
## Value Statement
As a Veteran
I want to be able to see that my claim has been closed and access my decision letter from the new V2 UI
So that I can view the status of my claim and decision

---
## Background Context

Design work in this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/75166

## Design

**Figma links**

- [Closed Claim](https://www.figma.com/file/F8U4wddaFouUPVd4mGBMDI/CST---Evidence-Submission---Spec?type=design&node-id=1124-8264&mode=design&t=uPIfXxmgwRwStjBC-0)

|         | Current Design | New Design |
| ------- | ------ | ----- |
| No Decision Letter | ![Screenshot 2024-02-28 at 8 55 12 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/ab1dc619-d4f9-47f5-92ee-27c70b179cde) | ![Screenshot 2024-02-28 at 12 23 58 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/a2a3090e-2d56-4b94-afad-5d0a366261bd) |
| With Decision Letter | ![Screenshot 2024-02-28 at 11 32 13 AM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/ca97de58-1bdb-4ec8-9566-1aa932ec6d2c) | ![Screenshot 2024-02-28 at 12 23 41 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/141954992/1c8e1629-b21e-4408-90a8-9158f54bfd3b) |

## Engineering 
- Add here

## Out of scope
- Add here

## Open questions
- Are we supposed to update the claim card? - No, the claim card is in the design just for context
- Should there be a payments section on the status tab? - yes , worked on design with skyler
- The AC says that there is always a blue alert when the claim is closed and another blue alert if no decision letter is available. The design only shows one blue alert though and it notes that the decision letter can be downloaded. Is there another design then that we are missing of just a closed alert? - discussed and updated design

## Tasks
- [x] An information alert displays with text and a link when a claim is closed with a decision letter sent
- [x] An information alert displays with text when a claim is closed with no decision letter sent
- [x] A "Payments" section is created and is only shown when a claim is closed
- [x] The "Last updated date" is hidden when claim is closed
- [x] The "Next Steps" section is shown when claim is closed
- [x] The "Recent Activity" section is shown when claim is closed
- [x] The "What you need to do" section is hidden when claim is closed
- [x] The "What we're doing" section is hidden when claim is closed
- [x] The "Claim Complete" component isn't displayed when cstUseClaimDetailsV2 feature toggle is enabled
- [x] The "Claims Decision" component isn't displayed when cstUseClaimDetailsV2 feature toggle is enabled
- [x] Design Check

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [x] Design performs design QA and verifies the implementation matches the design spec
- [ ] Engineering identifies staging users required to test and shares account and credentials with product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria

**Status Tab**
- [ ] When the claim is in Step 5/5 "Closed" and a decision letter is available, an information alert displays telling the Veteran that their claim has been closed with a link to the Claim Letters page
- [ ] When the claim is in Step 5/5 "Closed" and a decision letter is available, the "Get your claim letters" link  on the information alert takes the Veteran to the Claim Letters page
- [x] When the claim is in Step 5/5 "Closed" and no decision letter is available, an information alert displays telling the Vetran that a decision letter was mailed
- [ ] "Payments" section is shown when a claim is closed
- [ ] "Last updated date" is hidden when claim is closed
- [ ] "Next Steps" section is shown when claim is closed
- [ ] "Recent Activity" section is shown when claim is closed
- [ ] "What you need to do" section is hidden when claim is closed
- [ ] "What we're doing" section is hidden when claim is closed

**Comment by HeatherWidmont** [comment]
@samcoforma to review today

**Comment by pmclaren19** [comment]
@jacobworrell @skylerschain this has been merged! Please perform validation and Design QA and let me know if you have any questions!

**Comment by HeatherWidmont** [comment]
@skylerschain will review this today

**Comment by skylerschain** [comment]
Blocked by staging issues. Will come back to this.

**Comment by pmclaren19** [comment]
Hey @skylerschain this ticket was only supposed to cover the closed status on the 'Status Tab'. If you take a look at the ac you'll see that the things your noting are not a part of this ticket. The things that you are noting are actually a part of this other ticket that I havent started working yet [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77293). Please re-validate for Design QA, specifically only for the status tab/ the things noted in the ac above and if all looks good there I believe this is ready for @jacobworrell to validate. Let me know if you have any questions!

**Comment by skylerschain** [comment]
LGTM 👍 

Updated Tasks and DoD to indicate design checks are done.
